src/app/components/Header/index.tsx

```diff
@@ -157,6 +158,13 @@ export function Header() {
     },
     { to: '/stake', title: t(translations.mainMenu.staking) },
     { to: '/wallet', title: t(translations.mainMenu.wallet) },
+    {
+      to:
+        currentNetwork === 'mainnet'
```

Comment:
I would extract this logic to a variable so you don't have to duplicate it (something like `bridgeLink`).